install/ui/src/freeipa/ipa.js

```diff
@@ -251,7 +251,10 @@ var IPA = function () {
             on_success: function(data, text_status, xhr) {
                 that.whoami.metadata = data.result || data;
                 var wa_data = that.whoami.metadata;
-
+                // This AJAX request has no synchronization point,
+                // so we set async = false to make sure that init_metadata
+                // doesn't start before we get whoami response.
+                $.ajaxSetup({async: false});
```

Comment:
You don't need to set content or datatype and others here as well, ala install/ui/src/freeipa/config.js:91

Reply:
@rcritten 
There is a global ajaxSetup (freeipa/ipa.js:131), so this local setup overrides only one option and keeps other values which were specified previously.

Reply:
Ok, I wasn't sure if it replaced or augmented the settings. LGTM.